xfce-extra/multiload-nandhp/multiload-nandhp-0.2.ebuild

```diff
@@ -1,52 +0,0 @@
-# Copyright 1999-2015 Gentoo Foundation
-# Distributed under the terms of the GNU General Public License v2
-# $Header: /var/cvsroot/gentoo-x86/xfce-extra/multiload-nandhp/multiload-nandhp-0.2.ebuild,v 1.5 2015/07/01 08:09:08 zlogene Exp $
-
-EAPI=5
-inherit autotools eutils
-
-MY_PV=version-${PV/./-}
-
-DESCRIPTION="A port of the GNOME multiload applet for the Xfce and LXDE panels"
-HOMEPAGE="http://github.com/nandhp/multiload-nandhp"
-SRC_URI="http://github.com/nandhp/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
-
-LICENSE="GPL-2"
-SLOT="0"
-KEYWORDS="~amd64 x86"
```

Comment:
The old version is stable on `x86`, so you can't just remove it without getting the newer version tested and stabilized first. And anyway, it's often a good idea to delay the removal of old version after adding the new one.